src/fileimpl.cpp

```diff
@@ -105,21 +105,40 @@ sectionSubReader(const FileReader& zimReader, const std::string& sectionName,
         new DirectDirentAccessor(zimReader, std::move(urlPtrReader), entry_index_t(header.getArticleCount())));
 
 
-    auto titleIndexReader = sectionSubReader(*zimReader,
-                                        "Title index table",
-                                        offset_t(header.getTitleIdxPos()),
-                                        zsize_t(4*header.getArticleCount()));
-
-    mp_titleDirentAccessor.reset(
-        new IndirectDirentAccessor(mp_urlDirentAccessor, std::move(titleIndexReader), title_index_t(header.getArticleCount())));
-
     clusterOffsetReader = sectionSubReader(*zimReader,
                                            "Cluster pointer table",
                                            offset_t(header.getClusterPtrPos()),
                                            zsize_t(8*header.getClusterCount()));
 
     quickCheckForCorruptFile();
 
+    offset_t titleOffset(header.getTitleIdxPos());
+    zsize_t  titleSize(4*header.getArticleCount());
+
+    for (auto url:{"listing/titleOrdered/v1", "listing/titleOrdered/v0"}) {
```

Comment:
Why checking/trying "listing/titleOrdered/v0"? Isn't it a fallback via `header.getTitleIdxPos()` anyway?

Reply:
This is for coherence with the new spec.
The "new" way to locate the title ordered listing is to use the well-known path. The `header.getTitleIdxPos()` is the fallback.

But we can remove `listing/titleOrdered/V0` yes, not sure it is important anyway.